src/Stryker.Core/Stryker.Core/Instrumentation/StaticInitializerMarkerEngine.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.InjectedHelpers;
+
+namespace Stryker.Core.Instrumentation
+{
+    internal class StaticInitializerMarkerEngine: BaseEngine<ExpressionSyntax>
```

Comment:
I'm missing tests for this engine?

Reply:
no, the test is in MutantPlacerTests class